x-pack/plugins/security_solution/server/lib/timeline/saved_object/notes/saved_object.ts

```diff
@@ -81,6 +80,11 @@ export const getNotesByTimelineId = async (
   return notesByTimelineId.notes;
 };
 
+export interface InternalNoteResponse extends ResponseNote {
```

Comment:
Instead of maintaining this, would we want to update notes to use the pattern most of the other api's use? fyi @PhilippeOberti 

```
siemResponse.error({
            body: error.message,
            statusCode: error.statusCode,
          });
```

Reply:
This is only the internal representation which is in line with timeline.